.gitignore

```diff
@@ -0,0 +1,2 @@
+# Emacs temporary files
```

Comment:
i think these should go in your global .gitignore not the project-specific one

Reply:
Fair enough, and I can remove it if that is preferable.  I try to avoid using the global `.gitignore` in most cases, so that `git add .` is safe to use in general, and not just on my environment.  The tvm repo already has this line in its `.gitignore`, so I figured it was good to add here.  (Though in retrospect it probably should have been a separate PR.)

Reply:
okay, please remove this from the change then. we can debate it on another RFC